deeplabcut/pose_estimation_tensorflow/lib/inferenceutils.py

```diff
@@ -809,7 +809,14 @@ def wrapped(i):
                         if unique is not None:
                             self.unique[i] = unique
                         pbar.update()
-
+    
+    def from_pickle(self, pickle_path):
+        with open(pickle_path, "rb") as file:
+            data = pickle.load(file)
+        
+        self.assemblies = {key: value for key, value in data.items() if key != "single"}
```

Comment:
```python
self.unique = data.pop('single', {})
self.assemblies = data
```

Reply:
Done. A small annoyance is that it hangs a little when loading the `assemblies.pickle`, but I don't think this can be avoided

Reply:
No worries! That's nothing in comparison to the time that would otherwise be spent forming the assemblies.